code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -12,7 +12,7 @@
 	can_unwrench = FALSE
 	welded = FALSE
 	level = 1
-	layer = GAS_SCRUBBER_LAYER
```

Comment:
Any reason this was also changed?

Reply:
Scrubbers and the vents were moved up so they'll be above the resin.

Reply:
https://i.imgur.com/0fVnZ5t.png